rust/parquet/src/arrow/arrow_writer.rs

```diff
@@ -757,6 +820,25 @@ mod tests {
 
     #[test]
     #[ignore = "repetitions might be incorrect, will be addressed as part of ARROW-9728"]
```

Comment:
this `ignore` probably should be removed in this PR

Reply:
Removed, and the test passed :)